std_msgs/Header header
CupDetection[] detections
